shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -269,6 +330,44 @@ - (BOOL)isEqualTo:(FlutterTextRange*)other {
 }
 @end
 
+// A FlutterTextInputView that masquerades as a UITextField, and forwards
+// selectors it can't respond to to a shared UITextField instance.
+//
+// Relevant API docs claim that password autofill supports any custom view
+// that adopts the UITextInput protocol, automatic strong password seems to
+// currently only support UITextFields, and password saving only supports
+// UITextFields and UITextViews, as of iOS 13.5.
+@interface FlutterSecureTextInputView : FlutterTextInputView
```

Comment:
Changing `FlutterTextInputView` to a `UITextField` seems to require a lot of code changes so I chose this hack instead. From the documentation I think not supporting custom `UITextInput` is a bug in UIKit (unfortunately it happens on both iOS 12 & iOS 13), once it's fixed we can get rid of this class.

Reply:
Unrecognized selectors need to be forwarded to the shared text field instance because the keyboard implementation calls private `UITextField` methods (we tricked it into thinking this is a `UITextField` instance).